var/spack/repos/builtin/packages/rocm-debug-agent/package.py

```diff
@@ -11,27 +11,36 @@ class RocmDebugAgent(CMakePackage):
     """Radeon Open Compute (ROCm) debug agent"""
 
     homepage = "https://github.com/ROCm-Developer-Tools/rocr_debug_agent"
-    url      = "https://github.com/ROCm-Developer-Tools/rocr_debug_agent/archive/roc-3.5.0.tar.gz"
```

Comment:
You should keep this line (and update it to the new URL), it is used by `spack versions` and `spack checksum` to find new versions.